src/query/ee/src/storages/fuse/operations/vacuum_drop_tables.rs

```diff
@@ -77,6 +91,27 @@ async fn vacuum_drop_single_table(
                 error!("failed to remove all in directory {}: {}", dir, err);
             }
             result?;
+
+            for table_field in table_info.meta.schema.fields() {
+                if table_field.auto_increment_display().is_none() {
+                    continue;
+                }
+                let sequence_name = AutoIncrement::sequence_name(
+                    table_info.database_name()?,
+                    table_info.ident.table_id,
+                    table_field.column_id,
+                );
+
+                DropSequenceInterpreter::req_execute(
```

Comment:
Need add some comment why we need to drop sequence in here. cc @dantengsky  Please review this.

If this is a same logic with vacuum table ownership, should vacuum in here: https://github.com/datafuselabs/databend/blob/296e07247846a7d079da647fc721421995bc2b29/src/meta/api/src/garbage_collection_api.rs#L547